### DISCUSSION TITLE
Is there a workaround to setup Update/Create hooks for views?

### DISCUSSION THREAD

**Discussion by beltebelt** [pocketbase/pocketbase#4935] [Q&A]
I have a collection `posts` and I have setup a view `postsView` for the same collection which has some additional fields such as `repliesCount` and `likesCount` which is populated by joining `posts` with `replies` and `likes` table and getting the relevant count. This all works as expected. However, I would like to setup a hook which is triggered after a post is created or updated, but I want to use data from `postView` in this hook (I am exporting this data to a different app (meilisearch))

I could setup the AfterCreate/AfterUpdate hook and then fetch data from `postView` (`app.Dao().FindRecordById("postView", record.Id)`), but wanted to understand if there is a way to listen to the changes on the view itself.

**Comment by ganigeorgiev** [comment]
No. Views don't have create/update/delete events because views are practically "dynamic/computed" collections and they don't store or have data on their own (a View collection could be anything including just `select 1 as id`)

**Reply by beltebelt** [reply]
I see. I realized that I should be looking for changes in the counts of comments/likes using hooks on their respective tables. It has nothing to do with the post table anyway.

Thanks!
